examples/kivy/main.py

```diff
@@ -41,7 +41,9 @@ async def example(self):
         while self.running:
             try:
                 self.line("scanning")
-                scanned_devices = await bleak.BleakScanner.discover(1)
+                scanner = bleak.BleakScanner()
+                await scanner.request_permissions()
```

Comment:
This should only be called once at the start of the program. Also, it needs to be guarded with an if or in a try/except so that the program can run on platforms other than android.

Reply:
done